chirpus-2/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="nav">
+        <div class="nav__logo">chirpus inc.</div>
+        <div class="nav__links">
```

Comment:
I really like that you wrapped the nav links inside this parent div. You had the right idea, but you never targeted this class in your css file.